### PR TITLE
:bug: avoid error when format's params is empty

### PR DESCRIPTION
**Pull request by guillaumecrespel**
I've add a simple warning when the format's params is empty.
Beacause `intl-messageformat` throw a fatal error  😞

### CODE REVIEW

**Comment by guillaumecrespel** [comment]
what do you think to add console warning @fabienjuif ? 
in practice, this log is call many time 😥 
image

I think, it's not important to know that, no ?

File: src/format.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ export const formatter = (...args) => new IntlMessageFormat(...args)
 
 const formatMessage = (params, message = '', ...args) => {
   if (params) {
-    return formatter(message, ...args).format(params)
+    try {
+      return formatter(message, ...args).format(params)
+    } catch (e) {
+      console.warn('/ HOC k-intl /', e.message) // eslint-disable-line no-console
```
Comment (fabienjuif) [inline src/format.js]: In other lib we don't add extra spaces around `/ `

**Comment by fabienjuif** [comment]
I don't understand the issue.

If you don't pass all arguments this is cool to have some traces no ?

**Comment by guillaumecrespel** [comment]
at the moment, `int-messageformat` throw an exception when param is empty.
But when application loaded, you have not the param define.
by exemple on metroscope, the name or the version of project is set after fetch and the component header is loaded before.
So I thinked catch exception and log this on warning.
But it's not a realy warning i think, because it's a normal on life cycle of component.

**Comment by fabienjuif** [comment]
This is not normal.
You can add a `hoc-react-loader` before the intl take place

**Comment by guillaumecrespel** [comment]
ok, so it's ok for this PR ?

**Comment by fabienjuif** [comment]
Test what I say in Metroscope first.

**Comment by guillaumecrespel** [comment]
it's ok in metroscope, but so without `loader`: 
image
I don't know why I've this previous issue 😥

**Comment by fabienjuif** [comment]
Because you already have the locale (json) in you cache/before loading the page I think.

**Pull request merged**